html.go

```diff
@@ -124,7 +124,7 @@ func (w *HTMLWriter) Newline() {
 
 // NewHTMLRenderer creates and configures an HTMLRenderer object, which
 // satisfies the Renderer interface.
-func NewHTMLRenderer(params HTMLRendererParameters) Renderer {
+func NewHTMLRenderer(params HTMLRendererParameters) *HTMLRenderer {
```

Comment:
Sure.

Talking about the LaTeX renderer, it's currently a stub. Is there any plan for its implementation? If not, I'd be happy to work on it. I've already commited some changes to my fork (but that's for v1).